docs/pages/deploy-a-cluster/aws-gslb-proxy-peering-ha-deployment.mdx

```diff
@@ -0,0 +1,306 @@
+---
+title: "AWS Route 53 GSLB Multi-Region Proxy Peering High Availability Deployment Guide"
+description: "Deploying a Proxy-peered High Availability Teleport Cluster using Route 53 to create Global Server Load Balancing"
+---
+
+When deploying Teleport in production, you should design your deployment to
+ensure that users can continue to access infrastructure should an outage or 
+incident affect the availability of your Teleport cluster. 
+
+In order to maintain optimal end-user 
+experience with minimal latency and maximum performance, it is imperative to ensure the scalability 
+of your Auth Service and Proxy Service to accommodate increasing numbers of users and connected resources.
+
+(!docs/pages/includes/cloud/call-to-action.mdx!)
+
+## Overview
+This deployment architecure makes all connected resources accessible through a single Teleport cluster 
+across multiple regions using exclusively AWS ecosystem infrastructure. 
```

Comment:
I might move the note about "exclusively AWS ecosystem" down under the _Key deployment components_ section, since that's where you bring up all the AWS components.